force-app/main/default/customMetadata/APISettings.GitHubEndpoint.md-meta.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>GitHubEndpoint</label>
+    <protected>false</protected>
+    <values>
+        <field>Endpoint__c</field>
+        <value xsi:type="xsd:string">repos/pilivpp/ApexAdventures/commits?sha=MT-0001</value>
```

Comment:
[MAJOR]
NIe wiem czy to najlepszy pomysł trzymać w repo endpointy. Zrób MSa który tworzy ręcznie ten obiekt w systemie